platform/viewer-static/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ischemaview/ohif-viewer-static",
```

Comment:
can we call this `@ischemaview/rapid-dicom-viewer`?

Reply:
That is fine, but are we ok with keeping the non-static project name `@ischemaview/ohif-viewer`?

Reply:
Reverted `viewer` => `@ohif/viewer` for now, renamed `viewer-static` => `@ischemaview/rapid-dicom-viewer` as discussed